src/BuildScriptGenerator/Constants.cs

```diff
@@ -19,6 +19,7 @@ public static class Constants
         public const string ZipAllOutputBuildPropertyKeyDocumentation =
             "Zips entire output content and puts the file in the destination directory." +
             "Options are 'true', blank (same meaning as 'true'), and 'false'. Default is false.";
+        public const string EnumerateFilesInRepo = "Enumerating repo to find any files with extension '{0}'...";
```

Comment:
those strings should go in a labels resx file.

Reply:
Moved this string and the "Could not detect the language from repo." string to `Labels.resx` and replaced their occurrences with the new resource strings.